CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
+
+### Release Notes
+
+* Features:
+  * AKS version 1.30 is now available and will be the next LTS version of AKS. You can now [upgrade your 1.27 clusters](https://learn.microsoft.com/azure/aks/long-term-support#migrate-from-lts-to-the-next-lts-release) to 1.30 during the LTS period.
+  * [Updating an existing node pool to enable or disable FIPS](https://learn.microsoft.com/azure/aks/enable-fips-nodes#update-an-existing-node-pool-to-enable-or-disable-fips) is now Generally Available.
+  * AKS patch versions 1.30.3, 1.29.7, 1.28.12, 1.27.16 are now available. Refer to [version support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy) and [upgrading a cluster](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli) for more information.
+  * Istio add-on now only allows `EnvoyFilter`s of the types Lua, local rate limiting, and gzip compression.
+  * [Telemetry API v1](https://istio.io/latest/blog/2024/v1-apis/#telemetry-api) is now available for the Istio based service mesh add-on.
+  * The [AKS extension for Visual Studio Code](https://learn.microsoft.com/azure/aks/aks-extension-vs-code) now supports the ability to attach an ACR to your cluster, generate Kubernetes deployment files, generate Dockerfiles, and generate GitHub Actions.
+
+* Preview features:
+  * Dual-stack networking support for Azure CNI powered by Cilium is now in public preview.
+  
+* Bug fixes:
+  * Fixed a bug where sometimes `NodePublicIPPrefixID` could show unset on a cluster even though it was set.
+  * Previously, as part of Istio addon canary upgrade, users had to manually copy their edits to HorizontalPodAutoscaler from old revision to new revision. This has been fixed so that changes done to Horizontal Pod Autoscaler will be automatically copied for the newer revision.
+
+* Behavior change:
+  * When [Advanced Networking Observability](https://learn.microsoft.com/azure/aks/advanced-network-observability-concepts?tabs=non-cilium) is enabled, increased memory limit of 700Mi (from 400Mi) is used for retina-agent.
+  * `GOMAXPROCS` for coredns has been set to equal CPU limit to avoid throttling.
+  * In Azure CNI, `init-cni-dropgz` initContainer has been renamed to `cni-installer`.
+  * Validation for minimum 5 minutes has been introduced for [drain timeout value](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#set-node-drain-timeout-value) to prevent drain issues during upgrade.
```

Comment:
Creation of clusters with konnectivity and private Key Management Service (KMS) plugin based encryption of etcd using Azure Key Vault will not be supported since 19th August. Only clusters with API Server VNet Integration (preview) tunnel are allowed to be used along with KMS encrypted etcd clusters based on private Azure Key Vault.

Reply:
let's get internal confirmation on this

Reply:
per discussions on Teams, let's follow up after internal closure. we can update this release to add this line after that.